tools/cldr-apps/src/main/java/org/unicode/cldr/web/api/Summary.java

```diff
@@ -551,7 +561,7 @@ public Response getParticipationFor(
             coverageLevel = org.unicode.cldr.util.Level.fromString(level);
         }
         ReviewOutput reviewOutput =
-                new Dashboard().get(loc, target, coverageLevel, null /* xpath */);
+                new Dashboard().get(loc, target, coverageLevel, null /* xpath */, false);
```

Comment:
nit: never add a boolean without a comment what it is 
```suggestion
                new Dashboard().get(loc, target, coverageLevel, null /* xpath */, false /* includeOther */);
```

Reply:
I'm used to getting that filled in by the IDE but sounds good

<img width="498" alt="image" src="https://github.com/user-attachments/assets/cd2d2a15-0a6b-499f-8e5f-511a4a275d5e" />

Reply:
Ah! If you have a moment during the Stanford visit I'd love for your help setting up my IDE to for things like this. Do you use Eclipse or VSCode?

Reply:
VS code with the red hat, Java package